akka-stream/src/main/scala/akka/stream/impl/ActorRefBackpressureSinkStage.scala

```diff
@@ -40,25 +41,42 @@ import akka.stream.stage._
       var completeReceived = false
       var completionSignalled = false
 
-      private def receive(evt: (ActorRef, Any)): Unit = {
+      private def receive(ackMsg: Any)(evt: (ActorRef, Any)): Unit = {
```

Comment:
Givent that all the other logic is identical I think it would be nice to not duplicate and instead go with a guard covering both, something like `case ackMsg if ackMessage.isEmpty || ackMessage.contains(ackMsg) =>  ....`